packages/runner/package.json

```
@@ -4,6 +4,7 @@
   "private": true,
   "browser": "src/index.js",
   "scripts": {
+    "prebuild": "rimraf -rf ./dist",
```

```suggestion
    "prebuild": "rimraf ./dist",
```

I think the `-rf` part is inherent in `rimraf`